webapp/components/post_view/components/post_message_view.jsx

```diff
@@ -47,9 +56,30 @@ export default class PostMessageView extends React.Component {
         return false;
     }
 
+    isPostEdited() {
+        return this.props.post.update_at && this.props.post.update_at > this.props.post.create_at;
+    }
+
     render() {
+        const {formatMessage} = this.props.intl;
+
         if (!this.props.enableFormatting) {
-            return <span>{this.props.message}</span>;
+            return (
+                <span>
+                    {this.props.post.message}
+                    {
+                        this.isPostEdited() ?
+                            <span className='edited'>
+                                &nbsp;
+                                <FormattedMessage
```

Comment:
@enahum, If `this.props.enableFormatting` is true then the above JSX is rendered, but if it's false then the one at line 103 is rendered. And in the one at line 103, we are passing formatted message (formatted using `TextFormatting.formatText`) to `dangerouslySetInnerHTML` property and there I am using `formatMessage` (and using `injectIntl` and `defineMessages`) as I needed to append a HTML string to the HTML string returned by `TextFormatting.formatText`.